tests/components/accuweather/test_init.py

```diff
@@ -0,0 +1,59 @@
+"""Test init of AccuWeather integration."""
+from homeassistant.components.accuweather.const import DOMAIN
+from homeassistant.config_entries import (
+    ENTRY_STATE_LOADED,
+    ENTRY_STATE_NOT_LOADED,
+    ENTRY_STATE_SETUP_RETRY,
+)
+from homeassistant.const import STATE_UNAVAILABLE
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+from tests.components.accuweather import init_integration
+
+
+async def test_async_setup_entry(hass):
+    """Test a successful setup entry."""
+    await init_integration(hass)
+
+    state = hass.states.get("weather.home")
+    assert state is not None
+    assert state.state != STATE_UNAVAILABLE
+    assert state.state == "sunny"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)